go.mod

```diff
@@ -28,7 +28,9 @@ require (
 	github.com/inconshreveable/mousetrap v1.0.0
 	github.com/joho/godotenv v1.3.0
 	github.com/json-iterator/go v1.1.5
+	github.com/k0kubun/pp v3.0.1+incompatible
```

Comment:
This module is needed this PR? This is not imported in patch.

Reply:
oops.
It was a side effect of debugging. to erase.